### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by lordelph**
- recommended additional sleep after DNS self test
- added links to relevant external docs
- corrected typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,22 +122,36 @@ foreach ($authorizations as $authorization) {
...
+
+```php
+if (!$client->selfTest($authorization, Client::VALIDATON_DNS)) {
+    throw new \Exception('Count not verify ownership via DNS');
+}
+sleep(30); // this further sleep is recommended, depending on your DNS provider, see below
+``` 
+
+With DNS validation, after the `selfTest` has confirmed that DNS has been updated, it is 
+recommend you wait some additional time before proceeding, e.g. `sleep(30);`. This is
```
Comment (bakkerpeter) [inline README.md]: it is recommend**ed**

Comment (bakkerpeter) [reply README.md]: Thanks for fixing my typos, great additions. This will definitely help devs.

Comment (lordelph) [reply README.md]: corrected :)

**Comment by bakkerpeter** [comment]
Thanks!

**Pull request merged**